Wootric/WootricSDK-iOS
# Fix slider bug

Fix slider graphic bugs when selecting a score.

## Changes

- Fix slider bug
- Cleanup code

## Test

- Checkout this branch
- Check the slider is working correctly without the glitchy behavior reported in the issue

Issue: https://github.com/Wootric/WootricSDK-iOS/issues/82
